contrib/ibm-mq/Makefile

```diff
@@ -0,0 +1,15 @@
+TARGET=mq
```

Comment:
Makefile and Dockerfile remained after our development process in local test env. I'll remove them in favor of `ko`

Reply:
`ko` might not be the right tool here, if you need to use CGO and extract files from the redistributable object (due to licensing requirements).

Ideally, this would be handled by `dep` when you depend on the `mq-golang` package, but that may not be possible. In that case, figuring out how to release this using a Dockerfile may be the path of least resistance.